### PR TITLE
koordlet: add extended HTTP handler registry

### PR DESCRIPTION
**Pull request by saintube**
### Ⅰ. Describe what this PR does

koordlet: add a registry for external HTTP handlers.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Review by jasonliu747 — APPROVED** [review]
/lgtm

File: cmd/koordlet/main.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ import (
 	metricsutil "github.com/koordinator-sh/koordinator/pkg/util/metrics"
 )
 
+// ExtendedHTTPHandlerRegistry is the registry of extended HTTP handlers.
+var ExtendedHTTPHandlerRegistry = map[string]func() http.HandlerFunc{}
```
Comment (zwzhang0107) [inline cmd/koordlet/main.go:41]: /options dir？

Comment (saintube) [reply cmd/koordlet/main.go]: done

**Pull request merged**